src/components/layouts/sidebar/sidebar-bottom.tsx

```diff
@@ -45,98 +48,119 @@ export const SidebarBottom: FunctionComponent<React.PropsWithChildren<unknown>>
-							{version && (
-								<>
-									<p className="font-semibold text-white-high text-base mt-1">Network</p>
-									<p className="opacity-50 text-white-emphasis text-xs">{network}</p>
-								</>
-							)}
-						</div>
-					</div>
-					{!isMobileWeb ? (
-						<Button
-							backgroundStyle="primary"
-							onClick={e => {
-								e.preventDefault();
-								disconnectAccount();
-								disconnect();
-								queries.rebus.queryAccount.get(address).cancel();
-							}}>
-							<img alt="sign-out" className="w-5 h-5" src={`${MISC.ASSETS_BASE}/icons/sign-out-secondary.svg`} />
-							<p className="text-sm max-w-24 ml-3 overflow-x-hidden truncate transition-all">Sign Out</p>
-						</Button>
-					) : null}
-				</React.Fragment>
-			) : (
-				<ConnectAccountButton
-					style={{ marginBottom: '32px' }}
-					className="h-9"
-					textStyle={{ fontSize: '14px' }}
-					onClick={e => {
-						e.preventDefault();
-						connectAccount();
-					}}
-				/>
-			)}
-			<p className="py-2 text-xs text-white-high text-center opacity-30">
-				Price Data by
-				<a href="https://www.coingecko.com" target="_blank" rel="noreferrer">
-					{' CoinGecko'}
-				</a>
-			</p>
-			{/* <div className={'flex items-center transition-all w-full'}> */}
-			{/*<Img className="w-9 h-9" src={`${MISC.ASSETS_BASE}/icons/${openSidebar ? 'menu-in' : 'menu'}.svg`} />*/}
 			<div
 				className="flex items-center transition-all overflow-x-hidden w-full"
 				style={{ justifyContent: 'space-around' }}>
+				<button onClick={() => window.open(LINKS.MEDIUM)} className="opacity-75 hover:opacity-100 cursor-pointer mr-1">
+					<img
+						alt="medium"
+						style={{
+							width: '30px',
+							height: '30px',
+							filter: theme.text === '#FFFFFF' ? 'none' : 'invert(1)',
+							marginBottom: '3px',
+						}}
+						className="w-9 h-9"
+						src={`${MISC.ASSETS_BASE}/icons/medium.svg`}
+					/>
+				</button>
 				<button
 					onClick={() => window.open(LINKS.TWITTER)}
 					className="opacity-75 hover:opacity-100 cursor-pointer mb-0.5 mr-1">
 					<img
 						alt="twitter"
-						style={{ minWidth: '32px' }}
+						style={{ width: '24px', height: '24px', filter: theme.text === '#FFFFFF' ? 'none' : 'invert(1)' }}
 						className="w-8 h-8"
 						src={`${MISC.ASSETS_BASE}/icons/twitter.svg`}
 					/>
 				</button>
-				<button onClick={() => window.open(LINKS.MEDIUM)} className="opacity-75 hover:opacity-100 cursor-pointer mr-1">
-					<img
-						alt="medium"
-						style={{ minWidth: '36px' }}
-						className="w-9 h-9"
-						src={`${MISC.ASSETS_BASE}/icons/medium.svg`}
-					/>
-				</button>
 				<button
 					onClick={() => window.open(LINKS.DISCORD)}
 					className="opacity-75 hover:opacity-100 cursor-pointer mb-0.5">
 					<img
 						alt="discord"
-						style={{ minWidth: '36px' }}
 						className="w-9 h-9"
 						src={`${MISC.ASSETS_BASE}/icons/discord.svg`}
+						style={{ filter: theme.text === '#FFFFFF' ? 'none' : 'invert(1)' }}
```

Comment:
Please don't make comparisons like this, if you want to check if it's the light theme, use the current theme mode variable not the color of the text